### PR TITLE
Updates demo dependencies to avoid potential security threat

### PR DESCRIPTION
**Pull request by djvirgen**
>>>
djvirgen,
We found a potential security vulnerability in a repository for which you have been granted security alert access.

@ifwe	ifwe/monocle-api
Known high severity security vulnerability detected in mime < 1.4.1 defined in package-lock.json.
package-lock.json update suggested: mime ~> 1.4.1.

### CODE REVIEW

File: demo/package.json [hunk 1]
```diff
@@ -9,13 +9,13 @@
...
   "dependencies": {
-    "bluebird": "2.9.34",
-    "body-parser": "1.13.3",
-    "connect": "3.4.0",
-    "connect-chain": "0.0.1",
-    "lodash": "3.10.1",
-    "method-override": "2.3.5",
-    "serve-static": "1.10.2",
-    "swagger-ui-dist": "3.6.1"
+    "bluebird": "*",
```
Comment (TysonAndre) [inline demo/package.json]: Seems fine, could pin major versions instead to avoid accidentally upgrading to an incompatible version in the future.

Comment (DrakkhenD) [reply demo/package.json]: Yeah i was wondering what the implications would be for that

Comment (djvirgen) [reply demo/package.json]: Updated, locked in version numbers.

**Comment by DrakkhenD** [comment]
@hectorvirgen will this be merged?

**Pull request merged**